components/Msig/EnterOrCreateActor.jsx

```diff
@@ -100,7 +100,11 @@ const EnterActorAddress = () => {
           <br />
           <Box display='flex' flexDirection='row' alignItems='center'>
             <Text mr={3}>Don&apos;t have a multisig actor ID?</Text>
-            <StyledLink href='/vault/create?network=f' name='Create one' />
+            <StyledLink
+              href='/vault/create?network=f'
+              name='Create one'
+              target=''
```

Comment:
This is weird that we have to add this here to fix a bug. Can you please see this for context? https://github.com/glifio/wallet/commit/253705cc5043c6d1e82380467edf15485b259c91

Reply:
Short answer - set `target=_self`